_layouts/syllabus.html

```diff
@@ -55,6 +55,11 @@
       <a href="lectures/{{ page.slidesName }}.pdf">[pdf]</a>
       </p>
     {% endif %}
+    {% for oh_plus_plus in page.ohPlusPlus %}
+      <p>
+      <a href="/static/w17/oh-plus-plus/{{ oh_plus_plus }}">OH++ Slides</a>
```

Comment:
I am not sure what you mean by this, perhaps I am not experienced in GitHub rendering enough but I get a 404 error with this link and changed the week number as well.